### PR TITLE
[BREAK] Always remove the field `services` from user data responses in REST API

### PR DESCRIPTION
**Pull request by MarcosSpessatto**

### CODE REVIEW

**Comment by marceloschmidt** [comment]
@rodrigok @graywolf336 @MarcosSpessatto what's the status on this? we need to get it merged... has the mobile team been informed of the breaking change?

**Comment by geekgonecrazy** [comment]
@RocketChat/android @RocketChat/ios just in case not on radar.

File: packages/rocketchat-api/server/helpers/parseJsonQuery.js [hunk 1]
```diff
@@ -52,8 +60,12 @@ RocketChat.API.helperMethods.set('parseJsonQuery', function _parseJsonQuery() {
 	// Verify the user has permission to query the fields they are
 	if (typeof query === 'object') {
 		let nonQuerableFields = Object.keys(RocketChat.API.v1.defaultFieldsToExclude);
```
Comment (cardoso) [inline packages/rocketchat-api/server/helpers/parseJsonQuery.js]: "nonQueryable"

**Comment by cardoso** [comment]
@geekgonecrazy @marceloschmidt @MarcosSpessatto @graywolf336  this doesn't break anything for us

File: packages/rocketchat-api/server/helpers/parseJsonQuery.js [hunk 2]
```diff
@@ -22,8 +22,12 @@ RocketChat.API.helperMethods.set('parseJsonQuery', function _parseJsonQuery() {
 	// Verify the user's selected fields only contains ones which their role allows
 	if (typeof fields === 'object') {
 		let nonSelectableFields = Object.keys(RocketChat.API.v1.defaultFieldsToExclude);
-		if (!RocketChat.authz.hasPermission(this.userId, 'view-full-other-user-info') && this.request.route.includes('/v1/users.')) {
-			nonSelectableFields = nonSelectableFields.concat(Object.keys(RocketChat.API.v1.limitedUserFieldsToExclude));
+		if (this.request.route.includes('/v1/users.')) {
+			if (RocketChat.authz.hasPermission(this.userId, 'view-full-other-user-info')) {
```
Comment (ggazzo) [inline packages/rocketchat-api/server/helpers/parseJsonQuery.js]: what do you think
```javascript
	...
	const getFields = () => Object.keys(RocketChat.authz.hasPermission(this.userId, 'view-full-other-user-info') ? RocketChat.API.v1.limitedUserFieldsToExcludeIfIsPrivilegedUser : RocketChat.API.v1.limitedUserFieldsToExclude);
	nonSelectableFields = nonSelectableFields.concat(getFields());
	...
```

**Comment by marceloschmidt** [comment]
Can we merge this yet?

**Comment by MarcosSpessatto** [comment]
@ggazzo what do you think?

**Pull request merged**